ui/app/components/variable-paths.hbs

```diff
@@ -16,7 +16,7 @@
         <td colspan="3">
           <span>
             <FlightIcon @name="folder" />
-            <LinkTo @route="variables.path" @model={{folder.data.absolutePath}}>
+            <LinkTo @route="variables.path" @model={{folder.data.absolutePath}} @query={{hash namespace="*"}}>
```

Comment:
Since we're in the same route (and the parent route) is still activated, `LinkTo` will automatically forward the current query parameter. To override this behavior, we need to explicitly reset this parameter back to its original value `*`.